rtil/src/threads/ue/rebo/livesplit.rs

```diff
@@ -14,10 +16,107 @@ static VALID_SPLITS_PATHS: LazyLock<Mutex<HashSet<PathBuf>>> = LazyLock::new(||
 pub struct LiveSplit {
     pub timer: LiveSplitTimer,
     pub layout: LiveSplitLayout,
+    time: LiveSplitTime,
 }
 pub struct Timer {}
 pub struct Run {}
 pub struct Layout {}
+pub struct TotalPlaytime {}
+
+struct LiveSplitTime {
+    time: Time,
+    fraction: Fraction,
+}
+struct LiveSplitTimeInner {
+    time: TimeInner,
+    fraction: FractionInner,
+}
```

Comment:
It might make sense to move this into `livesplit-core` as well. (create 2 branches in your fork: one for getters/setters, one for this struct; make a PR to `livesplit-core` for each of your branches; combine your branches into `master` of your fork, use that in the `Cargo.toml` patch section)